README.md

```diff
@@ -62,7 +62,18 @@ bundle exec rspec spec/models
 
 # Run only specs for AccountsController
 bundle exec rspec spec/controllers/accounts_controller_spec.rb
+
+# Run only specs on a particular line. e.g line 14
+bundle exec rspec spec/controllers/accounts_controller_spec.rb:14
+
+# Run only specs that are tagged slow 
+bundle exec rspec spec/controllers/accounts_controller_spec.rb --tag slow 
+
+# Skip all specs that are tagged slow 
+bundle exec rspec spec/controllers/accounts_controller_spec.rb --tag "~slow"
+
```

Comment:
This is kind of arbitrary, theres a lot of different ways to run specs and this is still only a few.